web/src/javascript/drawMap.js

```diff
@@ -65,13 +65,25 @@ export function getPlayerColor(players, currentPlayer) {
 
 export function drawPlayersOnMap(players, canvas) {
   const context = canvas.getContext("2d");
+  let cellArray = [];
+  let cellInfo;
   for(const player of players) {
     const color = getPlayerColor(players, player);
     for(const cell of player.cells) {
-      drawCircle(context, cell.position, cell.radius, color);
-      writeCellTeamName(player.name, context, cell.position)
+      cellInfo = {
+        position: cell.position,
+        radius: cell.radius,
+        color: color,
+        playerName: player.name
+      };
+      cellArray.push(cellInfo);
     }
   }
+  cellArray.sort();
```

Comment:
i can do the second part but not the first because i have to have the color and the playername